### PR TITLE
Show hints when validating and submitting sentences (fixes #70)

### LINKED ISSUE #70: Show hint that sentences are being validated/uploaded
As uploading might take some time, we should show an indication that the sentences are being uploaded. We also might want to add a short notice that it might take some time depending on the amount of sentences.

Coming out of #46

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

File: web/src/components/submit-form.jsx [hunk 1]
```diff
@@ -44,9 +58,16 @@ export default class SubmitForm extends React.Component {
         </section>
 
         <section>
+          { parsingSentences && (
+            <p>
+              <b>Sentences are being validated. This can take a few seconds depending on the number of sentences added.</b>
+            </p>
+          )}
           <button>Submit</button>
```
Comment (MichaelKohler) [inline web/src/components/submit-form.jsx]: I suppose we should also disable the button when validating...

File: web/src/actions/parsing.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+  return {
+    existing: existingSentences,
+    valid: validNonExisting,
+    filtered,
+  };
+}
+
+async function getAlreadyDefinedSentences(language, sentences, credentials) {
+  const db = new WebDB(credentials.username, credentials.password);
```
[the remark this answers is not in the document]
Comment (freaktechnik) [reply web/src/actions/parsing.js]: To give a little more context, I'd use some memoization infrastructure like https://github.com/reduxjs/reselect/ if this is a rather slow process to cache the db. I don't think memoizing the whole function would yield performance benefits though.

[the remark this answers is not in the document]
Comment (MichaelKohler) [reply web/src/actions/parsing.js]: `default: 3ms` is what my `console.time()` says. I'll leave it as is, good catch though!

**Comment by MichaelKohler** [comment]
Yep, https://github.com/Common-Voice/sentence-collector/issues/102

**Pull request merged**